.github/workflows/release.yml

```diff
@@ -36,7 +35,7 @@ jobs:
       - name: Install dependencies
         run: yarn install --check-files --frozen-lockfile
       - name: release
-        run: npx projen release
+        run: yarn turbo run release --filter=functionless
```

Comment:
Release is run via turbo so all the references are resolved as if functionless was still the root.